### ISSUE TITLE
Update screenshots to match expectations

### ISSUE THREAD

**Issue by sotakoyama** [material-components/material-components-flutter-codelabs#287]
Thanks for the great tutorial. 
But i am facing one problem.

Buttons and app bars become transparent as shown in the image below.
Is there a way around this?

Development environment:
Windows 10
Flutter 3.7.10
Pixel 5a

開発環境：
Windows10
Flutter 3.7.10
Pixel 5a

![Screenshot_20230511-085804](https://github.com/material-components/material-components-flutter-codelabs/assets/91939052/4ec941ab-9cce-47d8-84a3-52a141e447b9)
![Screenshot_20230511-085855](https://github.com/material-components/material-components-flutter-codelabs/assets/91939052/064a21ec-3030-48f1-925b-1716e3e1bfc1)

**Comment by gbonesso** [comment]
Hi! I faced the same problem.
I'm using this codelab to teach Flutter in a high school. I just used this mistake it to encourage the students to search the API documentation and find a solution. Not planned, but it's worked well in my case. To get the same visual result just add some atributes:

`// TODO: Add app bar (102)

      appBar: AppBar(

        // TODO: Add buttons and title (102)

        title: const Text('SHRINE', 

          style: TextStyle(color: Colors.white)

        ),

        backgroundColor: Colors.blue,

      ),`

<img width="335" alt="image" src="https://github.com/material-components/material-components-flutter-codelabs/assets/11787066/fb81e904-9ec9-4136-8742-c20f238ef17c">
